source/extensions/extensions_metadata.yaml

```diff
@@ -1355,151 +1355,151 @@ envoy.matching.inputs.request_headers:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.SourceIPInput
 envoy.matching.inputs.source_port:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.SourcePortInput
 envoy.matching.inputs.direct_source_ip:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.DirectSourceIPInput
 envoy.matching.inputs.source_type:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.SourceTypeInput
 envoy.matching.inputs.server_name:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.ServerNameInput
 envoy.matching.inputs.transport_protocol:
   categories:
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.TransportProtocolInput
 envoy.matching.inputs.application_protocol:
   categories:
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.ApplicationProtocolInput
 envoy.matching.inputs.filter_state:
   categories:
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.FilterStateInput
 envoy.matching.inputs.uri_san:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.ssl.v3.UriSanInput
 envoy.matching.inputs.dns_san:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.ssl.v3.DnsSanInput
 envoy.matching.inputs.subject:
   categories:
   - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: alpha
+  status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.ssl.v3.SubjectInput
 envoy.matching.custom_matchers.trie_matcher:
   categories:
   - envoy.matching.http.custom_matchers
   - envoy.matching.network.custom_matchers
   security_posture: unknown
-  status: alpha
```

Comment:
FWIW we also need to update security posture to get Envoy security team coverage.

Reply:
I am wondering if it is ok we get this PR in and then have internal bug to track this effort since I feel it may take some time to do security review.

The context of this PR is i want to mark those components as stable to avoid breaking API changes. @adisuissa has caught one during API review recently

Reply:
Update: Synced with @yanavlasov this morning, I have created the internal tracker for this effort and will work with Kiriti on this. It may take some time as some extension usage touches parser.

Reply:
OK, but we do need to get that sorted :) Let's go with this for now.